opentelemetry-api/src/opentelemetry/baggage/propagation/__init__.py

```diff
@@ -63,7 +63,7 @@ def extract(
             )
             return context
 
-        baggage_entries = split(_DELIMITER_PATTERN, header)
+        baggage_entries: List[str] = split(_DELIMITER_PATTERN, header)
```

Comment:
Curious, upgrading requires us to add typing for defined variables now?

Reply:
Yes, this was necessary but I am not entirely sure why.